Day2/exercise1/solution.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+)
+
+var m = make(map[string]int)
+
+func freq_count(strin_channel <-chan string, strin_channel1 chan<- string) {
+	str_temp := <-strin_channel
+	x := len(str_temp)
+	for j := 0; j < x; j++ {
+		s := string(str_temp[j])
+		m[s]++
```

Comment:
multiple go routines can do operations on m. Handling of concurrent operations is not done.